src/groupeddataframe/splitapplycombine.jl

```diff
@@ -227,13 +227,15 @@ const F_ARGUMENT_RULES =
     * Column transformation operations using the `Pair` notation that is described below
       and vectors of such pairs.
 
-    Transformations allowed using `Pair`s follow the rules specified
-    for [`select`](@ref) and have the form `source_cols => fun`,
-    `source_cols => fun => target_col`, or `source_col => target_col`.
-    Function `fun` is passed `SubArray` views as positional arguments for each column
-    specified to be selected, or a `NamedTuple` containing these `SubArray`s if
-    `source_cols` is an `AsTable` selector. It can return a vector or a single value
-    (defined precisely below).
+    Transformations allowed using `Pair`s follow the rules specified for
+    [`select`](@ref) and have the form `source_cols => fun`, `source_cols => fun
+    => target_col`, or `source_col => target_col`. Function `fun` is passed
+    `SubArray` views as positional arguments for each column specified to be
+    selected, or a `NamedTuple` containing these `SubArray`s if `source_cols` is
+    an `AsTable` selector. It can return a vector or a single value (defined
+    precisely below). In particular if automatic generation of target column
```

Comment:
Why "in particular"?
```suggestion
    precisely below). If automatic generation of target column
```